tutorials/oauth2-with-23andMe-in-java/index.md

```diff
@@ -1,3 +1,11 @@
+---
+title: 23andMe API & google oauth2 in java
+description: round trip working example w/ sickle cell anemia
```

Comment:
Capitalize the description and provide a better text (e.g. `w/` is not appropriate). These show up in https://cloud.google.com/community/tutorials/, you can take a look at other tutorials for examples.